internal/services/keyvault/parse/sas_definition.go

```diff
@@ -15,6 +15,11 @@ type SasDefinitionId struct {
 	Name               string
 }
 
+func (i SasDefinitionId) ID() string {
+	fmtString := "%sstorage/%s/sas/%s"
```

Comment:
_minor nit:_ the Key Vault Base URI shouldn't end with a `/` - I'm going to push a commit to fix this (and in Key Vault's NestedItem too) - but this should otherwise be good to go 👍